src/components/common/modal/VideoModal.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+
+import { Modal, Button } from 'antd';
+
+function VideoModal() {
+  const [visible, setVisible] = useState(false);
+
+  const showModal = () => {
```

Comment:
Codacy found an issue: ['showModal' is assigned a value but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5801772)